lib/project_types/script/messages/messages.rb

```diff
@@ -70,9 +70,11 @@ module Messages
           help: <<~HELP,
           {{command:%1$s create script}}: Creates a script project.
             Usage: {{command:%1$s create script}}
-            Options:
-              {{command:--name=NAME}} Script project name. Any string.
-              {{command:--extension_point=TYPE}} Extension point name. Allowed values: %2$s.
+          HELP
+          extended_help: <<~HELP,
+            \s\sOptions:
```

Comment:
How come we need to add `\s`? It doesnt look like any other commands do this 🤔

Reply:
I did this to align Options with Usage. I think the difference between this and other commands is that if they are aligning things like this, they are in the same string and have some formatting that I couldn't seem to recreate without ```\s```. For example ```shopify help create``` has options and usage formatted this way, but they are both in the `rails.create.help` message. Open to alternatives to ```\s``` for sure though.

Reply:
Ah I see, I'm cool with that then!